microsoft-edge/devtools-guide-chromium/css/inspect.md

```diff
@@ -0,0 +1,82 @@
+---
+description: Learn how to use Microsoft Edge DevTools to analyze HTML pages using the Inspect tool 
+title: Inspect Tool in Microsoft Edge DevTools
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 12/16/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools
+---
+
+# Analyze HTML pages using the Inspect tool 
+
+This article shows you how to use the **Inspect** tool to preview information about and select an element from the current document. You can try it out yourself by checking the [Inspect Demo Page](https://microsoftedge.github.io/DevToolsSamples/inspector/inspector-demo.html).
+
+The **Inspect** tool button (![Inspect](../media/inspect-icon.msft.png)) is in the upper-left corner of DevTools.  When you select the **Inspect** tool button, the button turns blue, indicating that the **Inspect** tool is active.
+
+:::image type="complex" source="images/inspect-tool-button.msft.png" alt-text="The inspect tool button on the top right of the Developer Tools" lightbox="images/inspect-tool-button.msft.png":::
+   The inspect tool button on the top right of the Developer Tools
```

Comment:
This says "top right", but it should say "top left".

Reply:
fixed